packages/core/admin/admin/src/StrapiApp.js

```diff
@@ -228,8 +227,8 @@ class StrapiApp {
     }
 
     if (this.customConfigurations?.theme) {
-      const darkTheme = get(this.customConfigurations, 'theme.dark');
-      const lightTheme = get(this.customConfigurations, 'theme.light');
+      const darkTheme = this.customConfigurations.theme.dark;
+      const lightTheme = this.customConfigurations.theme.light;
```

Comment:
```suggestion
      const darkTheme = this.customConfigurations.theme?.dark;
      const lightTheme = this.customConfigurations.theme?.light;
```

Reply:
Since we have `if (this.customConfigurations?.theme)`, doesn't it mean that `theme` must be defined and non-null?

Reply:
I mean, is it possible to have a `theme` that is not an object anyway?

Reply:
True, my bad. But `?.light` and `?.dark` need to be optional, I think?

Reply:
I don't think they need to be, but I might be wrong!

Reply:
Yeah, I think if `theme` is defined and non-null, trying to parse properties that don't exist will never throw.
https://codesandbox.io/s/white-sun-bhohfm?file=/src/index.js

Reply:
Ok then - you know the code much better than me now :)